extensions/common/gcp/gcp-core/src/main/java/org/eclipse/edc/gcp/iam/IamServiceImpl.java

```diff
@@ -97,6 +99,45 @@ public GcpAccessToken createAccessToken(GcpServiceAccount serviceAccount) {
         }
     }
 
+    @Override
+    public GoogleCredentials getCredentials(GcpAccessToken accessToken) {
```

Comment:
please test this method

Reply:
At least one test that passes a `GcpAccessToken` and verifies that in output we have a `GoogleCredential` containing the token composed in a certain way, a classic unit test

Reply:
Thanks for the offline clarification, I have used mocks to test the method